### PR TITLE
New Medal: Party Hard.

### PR DESCRIPTION
**Pull request by Nebulacrity**
[FEATURE]

## About the PR 
Adds a "Party Hard" medal for dying while drunk either near neon lining or with a glowstick in your pocket.
ss13PartyHard
"You were the life of the party, before your own life got taken away."

## Why's this needed? 
More medals is always nice, and dying while vibing sounds achievement-worthy to me.

## Changelog

```
(u)Nebulacrity
(*)Added Party Hard medal for dying while drunk either near neon lining or with a glowstick in your pocket.
```

### CODE REVIEW

**Comment by Carbadox** [comment]
I like it alot but the criteria seems a bit strict, no? I feel like it would be better to make the requirements be "die drunk while being under neon lighting and/or with a glowstick on you.

**Comment by Nebulacrity** [comment]
Some medals are for moderately common actions, while others are for more obscure and/or complex ones. This is the latter.

**Comment by pali6** [comment]
Please provide a desciption and a medal picture to display on the byond medal list. Look [here](http://www.byond.com/games/Slurm/GoonStationMedals?command=view_available_medals) to see current medals.

**Comment by Nebulacrity** [comment]
Will do so in about 10 minutes.

**Comment by Nebulacrity** [comment]
Done.

File: code/mob/living/carbon/human.dm [hunk 1]
```diff
@@ -711,6 +711,12 @@
 			src.unlock_medal("Black and Blue", 1)
 		JOB_XP(src, "Clown", 10)
 
+		if (src.hasStatus("drunk"))
+			if(locate(/obj/item/device/light/glowstick) in src.contents)
+				for(var/turf/T in view(2, src.loc))
+					if(locate(/obj/neon_lining) in T.contents)
+						src.unlock_medal("Party hard", 1)
```
Comment (pali6) [inline code/mob/living/carbon/human.dm:718]: Capitalization should maybe be "Party **H**ard"?

Comment (Nebulacrity) [reply code/mob/living/carbon/human.dm:718]: Sure, I can fix that.

Comment (Nebulacrity) [reply code/mob/living/carbon/human.dm]: Done.

**Comment by Gerhazo** [comment]
The requirements feel too arbitrary/strict to me and this feels like a medal someone would have to deliberately read through the code to find out, rather than something they might do naturally or a reward for some more complex activity that someone would still naturally want to attempt or a rare game interaction.
I disagree with it being in the pile of the latter, because I doubt anyone would get the idea of doing what the medal entails, whereas the complex medals generally reward something difficult to do that still feels natural to try, or some very rare gameplay interactions.
Not to say that they're all perfect, as I can recall one medal that I'd consider similar in terms of arbitrary and that I can't remember if there are hints towards. I'd be happy with being DM'd any other potential ones that maybe I'm just failing to register.
Those are my thoughts, at least.

**Comment by Nebulacrity** [comment]
Guess I'll go with your suggestion then. Will commit soon.

**Comment by Nebulacrity** [comment]
Done.

**Comment by pali6** [comment]
Apparently I should merge it and someone (@mybluecorners ?) will add the medal?

**Pull request merged**